examples/data-objects/inventory-app/webpack.config.cjs

```diff
@@ -38,26 +36,20 @@ module.exports = (env) => {
 			output: {
 				filename: "[name].bundle.js",
 				path: path.resolve(__dirname, "dist"),
-				library: "[name]",
-				// https://github.com/webpack/webpack/issues/5767
-				// https://github.com/webpack/webpack/issues/7939
-				devtoolNamespace: "fluid-experimental/tree-api",
-				libraryTarget: "umd",
+				library: { name: "[name]", type: "umd" },
 			},
-			plugins: [
-				new webpack.ProvidePlugin({
-					process: "process/browser",
-				}),
-			],
```

Comment:
I believe a mistake in server code consumed by client used to cause this to be needed, but that was fixed and it works fine without it.